### DISCUSSION TITLE
局域网共享问题

### DISCUSSION THREAD

**Discussion by zer0cod3** [clash-verge-rev/clash-verge-rev#2429] [Q&A] [answered]
家里的MiniPC有线连接光猫，没有无线网卡，Clash Verge 开启局域网共享后，手机上能实现科学上网吗

![Screenshot 2024-12-22 224822](https://github.com/user-attachments/assets/a2b43973-8b50-4d68-80d8-b10d3c120a5b)
上图就是家里局域网设备情况，10是本机MiniPC，有线连接光猫，没有无线网卡。其他全是无线设备，5是我的手机

**Comment by zer0cod3** [comment, marked as answer]
搞定了

**Reply by tudan110** [reply]
请教一下，这个局域网连接怎么用啊
